src/mono/mono.proj

```diff
@@ -108,7 +108,7 @@
     Only wasi doesn't come with its own python and windows uses 32-bit python, so refer to the transport pack instead.
   -->
   <PropertyGroup Condition="'$(HostOS)' == 'windows' and '$(TargetsWasi)' == 'true'">
-    <PythonCmd>$([MSBuild]::NormalizePath('$(PkgMicrosoft_NET_Runtime_Emscripten_3_1_34_Python_win-x64)', 'tools', 'python'))</PythonCmd>
+    <PythonCmd>$([MSBuild]::NormalizePath('$(PkgMicrosoft_NET_Runtime_Emscripten_3_1_56_Python_win-x64)', 'tools', 'python'))</PythonCmd>
```

Comment:
I don't see an obvious way to keep the version string out of this one, the plan is to move this logic to emsdk properly where it is easier to do

cc @radekdoulik